### DISCUSSION TITLE
Winget UI does not recognize packages installed by previously Chocolatey

### DISCUSSION THREAD

**Discussion by Giamma94** [Devolutions/UniGetUI#1444] [Q&A] [answered]
Hello everyone, I have been using Chocolatey for a long time and have installed various packages with it.
Only recently I became aware of the existence of Winget UI.
I note with regret that Winget UI does not recognize packages that I had already installed from Chocolatey..
I don't know if this is normal, is there a solution?

Chocolatey GUI recognizes all packages without problems, but I would like to use Winget UI(I think it is done better).

Thank you if you answer me

**Comment by ContinuumOperand** [comment, marked as answer]
@Giamma94 Heya!

Try going to the WingetUI Settings menu, go in the **Package manager preferences** section, under **Chocolatey preferences** check **"Use system Chocolatey (Needs a restart)"**.


See of that helps in your case!

😊

WingetUI by default uses its own Chocolatey binary, this will use your previously uses, system installation of Choco instead.

**Reply by Giamma94** [reply]
Thank you very much for your reply!
I will try this evening, now I don't have my PC with me.
So you mean a reboot of the system not the SW, correct?

**Reply by ContinuumOperand** [reply]
I believe just the software is enough but at least a Logout can't hurt while at it.

**Reply by Giamma94** [reply]
As I thought, I have already done this test and it did not work.
I will try again this evening to make sure.

**Reply by ContinuumOperand** [reply]
Maybe @marticliment knows something else that can help in your situation.

**Reply by marticliment** [reply]
I will need to know:
1. Your windows version
2. Your chocolatey version
3. your WingetUI version

**Reply by Giamma94** [reply]
Thank you very much for the support, I solved it.
I am careless and didn't notice that Winget UI remains open as a service, closing it from the Try Bar and reopening it everything worked as it should.

Grazier so much for the support.

Do I close the question or delete it?

**Reply by marticliment** [reply]
No, it is fine to have it here so people with the same issue can benefit from it

**Reply by Spacellary** [reply]
I believe the question can stay for future reference and as a useful resource for more people.

**Reply by ppvnf** [reply]
maybe reference it in the FAQ
